components/platform_crypto/shave_trusted_boot/firmware.c

```diff
@@ -14,6 +15,240 @@
 #include "cn_array_utils.h"
 #define memcpy(f,b,s) _memcpy(f,b,s)
 #define memcmp(f,b,s) _memcmp(f,b,s)
+#define free _free
+#define realloc _realloc
+#endif
+
+//#define _Static_assert(...) extern int bogus
+#if !defined(NO_XMSS)
+#define union struct
+#define __STDC_NO_ATOMICS__ 1
+#include "endianness.h"
+#include "signing.h"
+#include "signing_private.h"
+#include "verification.h"
+#endif
+
+
+#if 0
```

Comment:
lets factor out the tests into a separate file, and remove this code. I think I saw a similar `if 0` in another file, and that one should be changed as well (but perhaps not in this PR)